account_bank_statement_import_crelan/readme/ROADMAP.rst

```diff
@@ -0,0 +1 @@
+Should be cleaned up and moved to https://github.com/OCA/bank-statement-import .
```

Comment:
Can you be more specific about what need to be cleaned ?

Reply:
done